app/models/manager_refresh/dto_collection.rb

```diff
@@ -15,6 +15,23 @@ def initialize(model_class, manager_ref: nil, attributes: nil, association: nil,
       @data         = []
       @data_index   = {}
       @saved        = false
+      @strategy     = process_strategy(strategy) || []
+    end
+
+    def process_strategy(strategy_name)
+      if strategy_name == :local_db_cache_all
+        process_strategy_local_db_cache_all
+      end
+      strategy_name
+    end
+
+    def process_strategy_local_db_cache_all
+      self.saved = true
+      selected = [:id] + manager_ref
+      selected << :type if model_class.new.respond_to? :type
+      parent.send(association).select(selected).find_each do |record|
```

Comment:
nice. not fully groking the implications of this selected line. but at a high level this sounds great